AUTHORS

```diff
@@ -0,0 +1,35 @@
+# List of AUTHORS who contributed over time to the AzerothCore project
+
+## Warning
+The code of AzerothCore is shipped as it is without any form of warranty,
+and - except for third party libraries licensed
+- under the GPL 2 for old files which you can read from the file "LICENSE-GPL2"
+- under the AGPL 3 for new files which you can read from the file "LICENSE-AGPL3"
```

Comment:
I think we can get rid of the double licensing as all GPLv2 source can be released with "GPLv2 or latter" version

Reply:
Okay

Reply:
Updated